src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
```

Comment:
call the method elsewhere in addressbook so that you can actually see the output!